src/requests/team.ts

```diff
@@ -19,6 +32,18 @@ export class TeamClient {
     return createTeam(this.client, args);
   }
 
+  async addMember(args: UpsertMember) {
+    return upsertMember(this.client, args);
+  }
+
+  async changeMemberRole(args: UpsertMember) {
+    return upsertMember(this.client, args);
+  }
```

Comment:
Using an upsert here means that "changeMemberRole" is not strictly only able to change an exiting member's role.

Calling it with a new unique combination of teamId/userId will be an INSERT operation but in reality this is going to be used by a handful of experienced users via the API docs for a finite time period, so I don't think this is too much of an issue.